scripts/ssh-into-vm.sh

```diff
@@ -1,134 +0,0 @@
-#!/usr/bin/env bash
-#
-# Convenience script to ssh into a particular VM using the local private key that was generated for
-# it.
-#
-# This script takes arguments from environment variables -- all optional. They are:
-#
-#   * VM_NAME - the vm.neon.tech/name of the VM. Optional if there is only one VM, otherwise must
-#     be provided.
-#   * SSHCLIENT_IP - the IP address of the ssh client on the vm-bridge network. Defaults to
-#     10.77.77.213, but must be provided if there is already a pod using that IP address.
-#   * NODE_NAME - if provided, a node to require that the pod is bound to. Used to allow access
-#     to the VM when inter-node networking is not working.
-
-set -e -o pipefail
-
-# Allow the script to be run from outside this directory
-cd -P -- "$(dirname -- "$0")"
-
-source '../scripts-common.sh'
-
-DEFAULT_SSHCLIENT_IP="10.77.77.213"
-SSHCLIENT_IP="$( ( set -u; echo "$SSHCLIENT_IP" ) 2>/dev/null || echo "$DEFAULT_SSHCLIENT_IP" )"
-
-NODE_NAME="$NODE_NAME" # because -u isn't set yet, this sets NODE_NAME to "" if it's undefined
-
-# can't enable `set -u` for get_vm_name because we might not have $VM_NAME
-vm_name="$(get_vm_name)"
-set -u
-
-SECRET_NAME='vm-ssh'
```

Comment:
Maybe it worth keeping the script up until this line, and replacing the rest with `kubectl exec $vm_name ssh ...`

That's basically what I did for qemu-monitor script.